src/Manager.js

```diff
@@ -81,20 +96,16 @@ class Manager {
 
   goToSection = (id) => {
     let element = this.anchors[id]
+    let viewHeight = this.config.container.innerHeight || this.config.container.clientHeight
+    let offset = this.config.offset + viewHeight/2
     if (element) {
-      jump(element, {
-        duration: this.config.scrollDuration,
-        offset: this.config.offset,
-      })
+      this.config.scroller.center(element, this.config.scrollDuration, offset)
```

Comment:
I'm confused about why use `scroller.center` + `this.config.offset + viewHeight/2` here?

If element's height is larger than container, it will not show the top of the element, but maybe we want to see the element from the beginning of it.

Maybe there is another solution? @dbramwell

Reply:
Did this 10 months ago, so the details are a bit fuzzy. If you look at the demo, it contains elements that are larger than the container and it does work as expected. I think I did it this way because of the way zenscroll works. From the docs:

"If you want you can also define an offset. The top of the element will be upwards from the center of the screen by this amount of pixels."

So, with an offset of zero, the top of the element would end up in the centre of the container. We want the top of the element to be at the top of the container so we need an offset of half the container's height (plus any offset requested in the config).

Does that answer the question?